polkadot-protocol/architecture/parachains/overview.md

```diff
@@ -5,7 +5,7 @@ description: Learn about the role, functionality, and implementation of parachai
 
 ## Introduction 
```

Comment:
Wow this is a long introduction. I think this is out of scope for this PR, but we need to revisit this page. I'll create a ticket.